test/support/test_hook.rb

```diff
@@ -0,0 +1,20 @@
+# typed: false
+# frozen_string_literal: true
+
+require "open_feature"
+
+class TestHook < OpenFeature::Hook::BeforeHook
```

Comment:
I was getting a lot of errors when trying simply to mock the classes that are "sorbet-conscious" if that's the right way to put it.

I resolved to this approach, although it is admittedly a little clunky.

Reply:
Yeah I believe we need only because we can't do runtime instantiation of abstract classes in Sorbet